client/js/lounge.js

```diff
@@ -485,7 +490,14 @@ $(function() {
 		container.html(templates.user_filtered({matches: result})).show();
 	});
 
-	var forms = $("#sign-in, #connect, #change-password");
+	searchQuery.on("click", ".show-more-button", function() {
+		$(this).text("Loading older messages…").prop("disabled", true);
+		var query = searchQuery.find("form").data("lastQuery");
```

Comment:
let

Reply:
Eslint advise to use const in that case, what's your opinion?

Reply:
We have a lot of old code that still has `var` unfortunately.